integration-tests/src/main/java/org/apache/druid/testing/clients/CoordinatorResourceTestClient.java

```diff
@@ -96,7 +95,7 @@ private String getLoadStatusURL()
   // return a list of the segment dates for the specified datasource
   public List<String> getMetadataSegments(final String dataSource)
   {
-    ArrayList<String> segments;
+    List<String> segments;
```

Comment:
This was required after upgrading Jackson to 2.10 (since `TypeReference<List<String>>` is used below)